metricbeat-tests/runner_test.go

```diff
@@ -180,7 +183,13 @@ func (mts *MetricbeatTestSuite) thereAreNoErrorsInTheIndex() error {
 
 	stackName := "metricbeat"
 
-	result, err := retrySearch(stackName, mts.IndexName, esQuery, queryMaxAttempts)
+	_, err := retrySearch(stackName, mts.IndexName, esQuery, queryMaxAttempts)
+	if err != nil {
+		return err
+	}
+
+	time.Sleep(runtimeDuration)
+	result, err := retrySearch(stackName, mts.IndexName, esQuery, 3)
```

Comment:
Should we use a constant value as max attempts here?

Reply:
Retry removed, this should work at this point.